poshi-runner/src/main/java/com/liferay/poshi/runner/PoshiRunnerContext.java

```diff
@@ -816,11 +817,12 @@ private static void _initComponentCommandNamesMap() {
 
 					_addComponentClassCommandNames(
 						componentName,
-						testCaseClassName + "#" + extendsCommandName);
+						testCaseNamespaceClassName + "#" + extendsCommandName);
 
 					_commandElements.put(
 						"test-case#" + _defaultNamespace + "." +
-							testCaseClassName + "#" + extendsCommandName,
```

Comment:
Is this right?  It looks like it's putting..
<pre>_defaultNamespace + "." + testCaseNamespaceClassName</pre>

as a key.  But that's 2 namespaces. right?

Reply:
You are correct, it shouldn't have 2 namespaces since testCaseNamespaceClassName already contains namespace (I changed it so namespaces are always stored with testcase names).  I will make the change along with any other questions you might have in another pull. Thanks!